pkg/eth/ipld_fetcher.go

```diff
@@ -167,8 +167,95 @@ func (f *IPLDFetcher) FetchRcts(tx *sqlx.Tx, cids []eth.ReceiptModel) ([]ipfs.Bl
 	return rctIPLDs, nil
 }
 
+// FetchLogs fetches logs.
+func (f *IPLDFetcher) FetchLogs(logCIDs []logResult) ([]*types.Log, error) {
```

Comment:
This isn't fetching anything, it's decomposing the already fetched logResults.

Reply:
Done